fuzz_testing/test/acl_globals_test.h

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2010-2021 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+
```

Comment:
I only see `acl_auto_configure_test.cpp` is being tested in the workflow run. Are other tests being fuzz tested as well? If they are not fuzz tested, we should remove them. If you have made changes to it and it's for the future release, you can move it to another draft PR.

Reply:
I believe the other test source code are included because at some point `acl_auto_configure_tet.cpp` called functions in other test cpp files. I have removed the other tests that `acl_auto_configure_tet.cpp` won't use.